src/pages/Invoice/InvoiceModal.js

```diff
@@ -6,18 +6,18 @@ import { FormattedMessage } from "react-intl";
 Modal.setAppElement("*");
 
 const ChangeCustomerModal = ({
-    modalStatus,
-    setModalStatus,
-    customersInfo,
-    itemInfo,
+    status,
+    setModalState,
+    customers,
+    items,
     type,
-    setState,
+    setFormState,
     fields,
 }) => {
     const updateCustomersDetails = useCallback(
         (name, phone, email) => {
-            setState("customers", { name, phone, email });
-            setModalStatus(false);
+            setFormState("customers", [{ name, phone, email }]);
+            setModalState((state) => ({ ...state, status: false }));
         },
```

Comment:
Can you please refer to the update str of the customer and make the needful changes

Reply:
The Str for customer have be changes to object from array of object